util/cron/test-linux64-llvm-bundled.bash

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+#
+# Test default configuration on examples only, on linux64, with bundled llvm
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/common.bash
+
+export CHPL_LLVM=bundled
```

Comment:
I believe this also needs `unset CHPL_LLVM_CONFIG`, otherwise the log is going to be filled with `warning: CHPL_LLVM_CONFIG is ignored with CHPL_LLVM=bundled`

Reply:
Good catch, thanks